projects/kit/src/lib/utils/is-empty.ts

```diff
@@ -0,0 +1,5 @@
+export function isEmpty(
```

Comment:
is this the expected behavior?

<img width="162" alt="image" src="https://user-images.githubusercontent.com/12021443/216959020-898c6f66-4f93-4119-8ba2-c3793a4e3cdc.png">

Reply:
It is falsy value, i expect it to return true.

Do you think I should remove primitives from the function signature ? 🤔 

```ts
export function isEmpty(
    entity: object | null | undefined,
): boolean {
```

Reply:
fixed ✅